README.md

```diff
@@ -200,19 +220,19 @@ settings.to_dict()
 # }
 ```
 
-## Валидация
-При переопределении метода `update_settings_from_source` рекомендуется использовать следующие методы класса `BaseSettings`:
+## Validation
 
-1. `pre_validate` — проверка наличия типов в `Property`, проверка соответствия типу `Property` в возможных значениях `Property.choices`, проверка значения по умолчанию на соответствии типу.
+It is recommended to use following `BaseSettings` class methods during redefinition `update_settings_from_source` method:
 
-2. `post_validate` — проверка на то, что каждому `Property` присвоено какое-либо значение
+1. `pre_validate` - check for presence of types in `Property`, check for compliance with the type `Property` in possible values ​​of `Property.choices`, check for default values ​​for type matching.
```

Comment:
check that types are configured correctly; check that the values from `choices` and the `default` pass the type check.